src/lib.js

```diff
@@ -41,6 +43,44 @@ var curlConverter = {
         return str;
     },
 
+    addQueryParamsFromDataOption: function(curlObj, urlData, request) {
+        // What this will do:
+        // If URL is http://example.com?a=b and -d 'c=d' => http://example.com?a=b&c=d
+        // If URL is http://example.com#fragment and -d 'c=d' => http://example.com#fragment
+        if ((curlObj.uploadFile.length > 0 || curlObj.head || curlObj.data.length > 0) && curlObj.get) {
```

Comment:
Can you mention why are we checking for curlObj.get ? @Dhroov7

Reply:
Need to check for behavior if `-I and -d` are both provided. curl thrown an error.

Reply:
@Dhroov7 Any updates on this?
Need jsdoc explaining the params
also, need to explain this check better